1_fetch/src/munge_reach_attr_tbl.R

```diff
@@ -0,0 +1,53 @@
+munge_reach_attr_table <- function(prms_reach_attr_csvs){
+  #'
+  #' @description Function to read in the PRMS reach attribute table and add 
+  #' _1 PRMS reaches for those reaches with _2 split segments.
+  #' 
+  #' @param prms_reach_attr_csvs p1_prms_reach_attr_csvs with a csv that 
+  #' includes reach_attributes in the filename
+  #' 
+  #' @value PRMS reach attribute table with added _1 reaches for _2 segments.
+  
+  # PRMS reach attribute table
+  attr_tbl <- read_csv(grep("reach_attributes", prms_reach_attr_csvs, value=TRUE),
+                       show_col_types = FALSE)
+  
+  # Special handling to add _1 split segments to PRMS segments with _2 suffix
+  # reach attribute table; values for from_segs and to_seg were
+  # derived based on visual inspection and manually entered
+  attr_tbl_w_splitsegs <- attr_tbl %>%
+    mutate(to_seg = as.character(to_seg),
+           subseg_seg = as.character(subseg_seg)) %>%
+    tibble::add_row(subseg_id = "3_1", from_segs = "1", subseg_seg = "3_1", 
+                    to_seg = "3_2", seg_id_nat = 1437, subseg_length = 14212.94) %>%
+    tibble::add_row(subseg_id = "8_1", from_segs = NA, subseg_seg = "8_1", 
+                    to_seg = "8_2", seg_id_nat = 1442, subseg_length = 17676.98) %>%
+    tibble::add_row(subseg_id = "51_1", from_segs = "43", subseg_seg = "51_1", 
+                    to_seg = "51_2", seg_id_nat = 1485, subseg_length = 105.14) %>%
```

Comment:
I see you can get the to seg and from_segs via inspection of the catchments. Wondering - how did you calculate the length and and how did you get the seg_id_nat for these specific PRMS_segid?

Reply:
I copied the length (subseglen) and seg_id_nat from `p1_reaches_sf`. I'd normally add `p1_reaches_sf` as a function argument in a case like this, but this whole function is hard coded, so it didn't seem necessary. _1 split reaches have NA seg_id_nat, so I assigned the value for the _2 segment to the _1 segment. I'll add comments for these details.